README.md

```diff
@@ -146,5 +163,7 @@ Kube Scheduler           | Watches Kube scheduler
 Ingress                  | Watches Routers                                                                                    | :heavy_check_mark:        |
 Openshift SDN            | Watches SDN pods                                                                                   | :heavy_check_mark:        |
 OVNKubernetes            | Watches OVN pods                                                                                   | :heavy_check_mark:        |
+Node Problem Detector    | Watches Node Problem Detector pods                                                                 | :heavy_check_mark:        |
```

Comment:
Let's just have the default namespaces being monitored ( system critical services ) in this table. We can skip node problem detector since it's optional.